MovieApp/Views/HomeScreen/FocusedHomeScreen/FocusedHomeTableViewCell.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  focusedHomeTableViewCell.swift
+//  MovieApp
+//
+//  Created by Nikola Đokić on 15.04.2022..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+import MovieAppData
+
+class FocusedHomeCollectionViewCell: UICollectionViewCell{
+    
+    static let id = "FocusedHomeCollectionCell"
+    private var moviePoster: UIImageView!
+    private var movieTitle: UILabel!
+    private var movieDescription: UILabel!
+    private var shadow: UIView!
+    private var shapes: UIView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    required init?(coder: NSCoder){
+        fatalError("Init(coder:) has not been implemented")
+    }
+    
+    public func configureMovie(movie: MovieModel){
+        shadow = UIView()
+        self.contentView.addSubview(shadow)
+        
+        shapes = UIView()
+        self.contentView.addSubview(shapes)
+        
+        createShadows()
+        
+        let data = try? Data(contentsOf: URL(string: movie.imageUrl)!)
```

Comment:
This could very easily fail and since you marked it with try? the `data` will be nil if that happens, and your app will crash in the next line. Just try it without network connection and you'll see. Use `guard` to handle that scenario.